### PR TITLE
CIF-1027 - Clean up occurrences of ccif in commerce-cif-connector project

### PR DESCRIPTION
**Pull request by LSantha**
There are several occurrences of the old acronym CCIF in commerce-cif-connector as "ccif" in component names and variable names which is now obsolete and misleading and it should be replaced with cif.

## Description

I've replaced occurrences of ccif with cif.

## Related Issue

CIF-1027

## Motivation and Context

Consistent naming, fixing deprecated naming patterns. 

## How Has This Been Tested?

Manually.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: content/cif-connector/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <packaging>content-package</packaging>
     
     <name>CIF Connector - Content Package</name>
-    <description>Content Package of Connector for CCIF hosted on openwhisk</description>
+    <description>Content Package of Connector for CIF hosted on openwhisk</description>
```
Comment (cjelger) [inline content/cif-connector/pom.xml]: Please update the description, this is no longer "hosted on openwhisk".

File: content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html [hunk 2]
```diff
@@ -18,9 +18,9 @@
         <coral-columnView-preview-label>${'Title' @ i18n}</coral-columnView-preview-label>
         <coral-columnview-preview-value>${helper.title}</coral-columnview-preview-value>
 
-        <sly data-sly-test.ccifId="${helper.properties.ccifId}">
+        <sly data-sly-test.cifId="${helper.properties.cifId}">
```
Comment (cjelger) [inline content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html:21]: Shouldn't we have an equivalent change to rename that property somewhere?

File: content/cif-connector/pom.xml (hunk 1 above)
Comment (LSantha) [reply content/cif-connector/pom.xml]: Any proposal for a new description?

File: content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html (hunk 2 above)
Comment (LSantha) [reply content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html:21]: Curiously enough we had already cifId in the properties and it was not displayed, so this fixes a hidden bug as well.

File: content/cif-connector/pom.xml (hunk 1 above)
Comment (cjelger) [reply content/cif-connector/pom.xml]: Just remove "hosted on openwhisk" and it's fine ;-)

File: content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html (hunk 2 above)
Comment (cjelger) [reply content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html:21]: We should then probably add a test for that, I think we only have one for the category properties page (https://github.com/adobe/commerce-cif-connector/blob/master/it/http/src/test/java/com/adobe/cq/commerce/it/http/GraphqlProductConsoleIT.java#L225)

Comment (cjelger) [reply content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html:21]: (we can add that test later, didn't mean it for this PR)

**Review by cjelger — APPROVED** [review]

**Pull request merged**